benchmark-service/benchmark/benchmark.py

```diff
@@ -0,0 +1,58 @@
+import os
+from pathlib import Path
+from typing import Any, AsyncIterator, Dict
+
+import aiohttp_jinja2
+import jinja2
+from aiohttp import web
+import logging
+
+app = web.Application()
+router = web.RouteTableDef()
+logging.basicConfig(level=logging.DEBUG)
+
+
+aiohttp_jinja2.setup(
+    app, loader=jinja2.ChoiceLoader([
+        jinja2.PackageLoader('benchmark', 'templates')
```

Comment:
You don't need the `'templates'` argument. That's the default with the `PackageLoader`.